.circleci/config.yml

```diff
@@ -6,16 +6,19 @@ jobs:
       - image: python:3.8
     steps:
       - checkout
-      - restore_cache:
+      - &restore_venv_cache
+        restore_cache:
           keys:
             # when lock files change, use increasingly general
             # patterns to restore cache
             - &cache_key
-            # yamllint disable-line rule:line-length
-             python-packages-v1-{{ .Branch }}-{{ checksum "requirements.in" }}-{{ checksum "requirements.txt" }}
-            - python-packages-v1-{{ .Branch }}-{{ checksum "requirements.in" }}-
-            - python-packages-v1-{{ .Branch }}-
-            - python-packages-v1-
+              # yamllint disable-line rule:line-length
+              python-3.8-packages-v1-{{ .Branch }}-{{ checksum "requirements.in" }}-{{ checksum "requirements.txt" }}
```

Comment:
this had been broken because the cache was poisoned by symlinks from when it was for python 3.7, and python packages are installed to a version-specific `site-packages` directory.